File: PB006.cs/Algorithm.cs
using System;

namespace ProjectEuler
{
    public class Algorithm : IAlgorithm
    {
        public string Compute()
        {
            long SumOfSquare=0, SquareOfSum=0;
            for (int i = 0; i <= 100; i++)
            {
                SumOfSquare += i * i;
                SquareOfSum += i;
            }
            SquareOfSum *= SquareOfSum;
            return (SquareOfSum - SumOfSquare).ToString();
        }

        public bool Prepare()
        {
            return true;
        }
    }
}

File: PB006.fs/Algorithm.fs
namespace ProjectEuler

type public Algorithm() = class
    let square x = x * x

    let SumOfSquare n = 
        [1..n]
        |> List.map square
        |> List.sum

    let SquareOfSum n = 
        [1..n]
        |> List.sum
        |> square

    let ans n = (SquareOfSum n) - (SumOfSquare n)

    interface IAlgorithm with
        member this.Compute(): string = ans 100 |> string
        member this.Prepare(): bool = true
    end
end
